queue_services/account-mailer/src/account_mailer/email_templates/common/style.html

```diff
@@ -4,8 +4,8 @@
    */
   @font-face {
     font-family: 'BCSans-Regular';
-      src: url('https://minio.apps.silver.devops.gov.bc.ca/public/BCSans-Regular.woff2') format('woff2'),
-          url('https://minio.apps.silver.devops.gov.bc.ca/public/BCSans-Regular.woff') format('woff');
+      src: url('https://storage.googleapis.com/auth-font-store/BCSans-Regular.woff2') format('woff2'),
```

Comment:
I wonder if it's possible to use these locally instead?

Reply:
@pwei1018 do we want to have a central location for fonts/images in a dedicated google bucket? or do we want to bundle them in the app, e.g. like https://github.com/bcgov/bcros-common/blob/main/document-service/documents-ui/src/assets/styles/base.scss
pros for using bucket:
   - smaller image (faster startup times)
   - easier to maintain across other repos (will need to switch over)
pros for bundling:
   - latency reduced?

Reply:
I think move it back to application should be good.